_posts/en/0500-10-24-tm-admin_en.md

```diff
@@ -13,30 +13,34 @@ category: coordination
+The Tasking Manager is the essential tool for all kind of groups to coordinate mapping on OpenStreetMap. It helps you to conduct a mapathon, manage a humanitarian mapping initiative, or whenever you want to create mapping tasks for students. The application divides an area of interest into manageable geographic chunks, that can be completed rapidly and collaboratively. This approach allows the distribution of tasks to many individual mappers, while reducing possible editing conflicts. 
 
-The Software further supports mapping accuracy and data quality by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities. This module describes the basic administration of the  Tasking Manager for successful mapping events.
+The Software further supports mapping accuracy and good data quality by providing a consistent set of instructions for your mappers (e.g. 'map all roads and buildings'). In short, the Tasking Manager is how you set up and direct the workflow for open mapping activities. 
 
-This guide covers instructions on the administration of the Tasking Manager, including the creation and modification of mapping projects and handling permissions for mapping and review and validation of data.
+This guide describes the basic administration of the Tasking Manager. It covers instructions to create and modify mapping projects and handling permissions for mapping and review of crowd-sourced data.
 
-## Access the Tasking Manager
+## Login and access
 
-Access is given to the Tasking Manager by users authorizing themselves through the [OpenStreetMap website](https://www.openstreetmap.org). In order to login to the Tasking Manager an OpenStreetMap (OSM) account is needed. You can visit the [HOT Tasking Manager](http://tasks.hotosm.org), or any other community instance of your preferance. There click `Sign up` and it redirects you to OpenStreetMap.org to create your account. Or click `Log in` if you have already one, and in the pop-up you can give the Tasking Manager limited access to your OSM account. 
+Access is given by users authorizing themselves through the [OpenStreetMap website](https://www.openstreetmap.org). In order to login to the Tasking Manager an OpenStreetMap (OSM) account is needed. You can visit the [HOT Tasking Manager](http://tasks.hotosm.org), or any other community instance of your preferance. There click `Sign up` and it redirects you to OpenStreetMap.org to create your account. Or click `Log in` if you have already one, and in the pop-up you can give the Tasking Manager limited access to your OSM account. 
 
```

Comment:
Suggestion: 
In order to login to the Tasking Manager an OpenStreetMap (OSM) account is needed. You can visit the [HOT Tasking Manager](http://tasks.hotosm.org), or any other community instance of your preferance. There click `Sign up` and it redirects you to OpenStreetMap.org to create your account. Or click `Log in` if you have already one, and in the pop-up you can give the Tasking Manager limited access to your OSM account.